Fails to build on react-native 0.73

Hey.
`react-native-dialogs` fails with the following error:

```
<project>/node_modules/react-native-dialogs/android/src/main/java/com/aakashns/reactnativedialogs/modules/DialogAndroid.java:457: error: cannot find symbol int dp = (int) (getReactApplicationContext().getResources().getDimension(R.dimen.md_listitem_height) ^ symbol: variable dimen location: class R
```

Code is:

https://github.com/react-native-dialogs/react-native-dialogs/blob/60b78c8750c26f5c4d6f09fe409d28c5a0d04253/android/src/main/java/com/aakashns/reactnativedialogs/modules/DialogAndroid.java#L457-L458

Presumably this is due to `md_listitem_height` not being read from `com.afollestad.materialdialogs`.

Pull request fix for this here: https://github.com/react-native-dialogs/react-native-dialogs/pull/143

--- a/android/src/main/java/com/aakashns/reactnativedialogs/modules/DialogAndroid.java
+++ b/android/src/main/java/com/aakashns/reactnativedialogs/modules/DialogAndroid.java
@@ -9,7 +9,7 @@ import android.view.View;
 import android.os.Build;
 import android.view.WindowManager;
 
-import com.aakashns.reactnativedialogs.R;
+import com.afollestad.materialdialogs.R;
 import com.afollestad.materialdialogs.DialogAction;
 import com.afollestad.materialdialogs.GravityEnum;
 import com.afollestad.materialdialogs.MaterialDialog;

